tidy3d/components/simulation.py

```diff
@@ -3638,10 +3638,7 @@ def snap_to_grid(geom: Geometry, axis: Axis) -> Geometry:
                 temp_geometry = set_bounds(snapped_geometry, bounds=new_bounds, axis=axis)
```

Comment:
I'm not sure about this line (and the line above):
```
new_bounds = (snapped_center - dls[0] / 2, snapped_center + dls[1] / 2)
temp_geometry = set_bounds(snapped_geometry, bounds=new_bounds, axis=axis)
```
The bound here is still of dl-thickness. Is that the right approach @dmarek-flex ?